### DISCUSSION TITLE
Attach video (upload) should really provide a progress percentage

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43968] [Mobile] [answered]
Attaching a 10mb video triggers an indefinite spinner. It's really important for large transactions to give a sense of overall progress --
* Will my battery run out before this upload finishes?
* Should I wait before hopping into a subway that will take me out of cell service? 

![Screenshot_20230111-095604.png](https://user-images.githubusercontent.com/2119212/211839296-a868bab3-413e-4bf4-b79a-ec47a5e18427.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref we can look into adding a progress bar or some sort of indication. Thanks!

**Reply by jsoref** [reply]
Sticking a percentage number into the circle would be a pretty reasonable approach
<img width="199" alt="image" src="https://user-images.githubusercontent.com/2119212/211910913-1261e3dd-9c7a-408d-9416-c5254d8345fa.png">
